src/test-tree/createTests.ts

```diff
@@ -222,37 +228,64 @@ export class TestCase {
 		readonly package_name: string,
 		readonly proof_boolean: boolean,
 		readonly stubbing_request?: boolean,
+		readonly module_name?: string,
 	) {}
 
 	getLabel(): string {
 		return `${this.harness_name}`;
 	}
 
+	// Get the fully qualified name for a harness from the file name, module name and harness name
+	getFullyQualifiedName(): string {
```

Comment:
For now, I recommend that you keep this logic. I would rename this function to something like, `expandFunctionName`, and add a comment:
```
// Expand the harness name to include the name of the file and module name as an attempt to narrow down the number of matches for this harness.
// This is a heuristic, and it may generate an identifier that does not match any symbol.
```